tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/AmqpUri.java

```diff
@@ -86,4 +86,22 @@ private RabbitMQConfiguration.ManagementCredentials parseUserInfo(String userInf
                 .join(passwordParts)
                 .toCharArray());
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Shall be `final`. 

(Is this tested by equals-verifier?)

Reply:
No, not yet. I'm currently testing AmqpURI but haven't pushed the changes yet because I'm encountering some issues with the URI class in the Java standard. For instance, BAD_URI is considered a valid URI and does not throw an exception.